src/cow-react/modules/limitOrders/pure/ReceiptModal/DateField.tsx

```diff
@@ -1,17 +1,21 @@
-import moment from 'moment'
 import * as styledEl from './styled'
+import useTimeAgo from 'hooks/useTimeAgo'
+import { format } from 'date-fns'
 
 interface Props {
-  date: string | Date | undefined
+  date: Date | undefined
 }
 
 export function DateField({ date }: Props) {
+  const timeAgo = useTimeAgo(date)
+  const previewDate = format(date || 0, 'MMM d YYY, h:mm a')
```

Comment:
Would [this](https://github.com/cowprotocol/cowswap/blob/7dc13801002e667e2625f4d51d30cf294c0d483f/src/cow-react/utils/time.ts#L15) help here?

Reply:
I don't think so, if we want this specific format

Reply:
Why is this case we want a specific date format?